src/pages/sok.tsx

```diff
@@ -0,0 +1,213 @@
+import { useLocale } from "@/locales";
```

Comment:
We can still scroll outside the block:

<img width="1433" alt="image" src="https://github.com/datasektionen/ddagen/assets/66337956/3542adac-627b-43c1-bc0e-bdb704b49ce1">

Reply:
Scrolling is now disabled when a team is clicked on, thereby not making it posible to scroll outside the block